recipe/meta.yaml

```diff
@@ -17,17 +17,16 @@ source:
   sha256: 7df0f84c4b6210112353374a7d992bfd5bba9c97358b39ea03b61bd634ce9c8a
 
 build:
-  number: 2
-  # TODO: Add `linux_aarch64`, `linux_ppc64le`, and `win_64` once supported on CUDA 12 in conda-forge.
-  # xref: https://github.com/conda-forge/cuda-python-feedstock/issues/36
-  # xref: https://github.com/conda-forge/cuda-python-feedstock/issues/37
-  # xref: https://github.com/conda-forge/cuda-python-feedstock/issues/38
-  skip: true  # [win or osx or cuda_compiler_version != "12.0"]
+  number: 3
+  # Cython 0.29.x seems not to be fully compatible with Python 3.12, skipping for now.
+  # This can be removed in a later CUDA Python that supports Cython 3.x.
+  skip: true  # [osx or cuda_compiler_version != "12.0" or py == 312]
```

Comment:
```suggestion
  # Cython 0.29.x seems not to be fully compatible with Python 3.12, skipping for now.
  # This can be removed in a later CUDA Python that supports Cython 3.x.
  # xref: https://github.com/conda-forge/cuda-python-feedstock/issues/57
  skip: true  # [osx or cuda_compiler_version != "12.0" or py == 312]
```